scripts/sv/default_init.sh

```diff
@@ -18,10 +18,7 @@ ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
 REFLOC=$(/usr/share/google/get_metadata_value attributes/reference)
 SAMPLE=$(/usr/share/google/get_metadata_value attributes/sample)
```

Comment:
I believe the comment are outdated now, as don't do anything now for the master.
And it would be better if we remove the unused variables `NAME`, `SAMPLE`.